.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+<!-- Thanks for contributing to _OHHTTPStubs_! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] -->
+
+### Checklist
```

Comment:
Please add an empty newline after markdown section titles (it's my OCD talking 😄)